java/cleanJdk8.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "$1" == "8" ]; then
```

Comment:
Why don't make that in the Dockerfile?

Reply:
Because the multiline if else didn't seem to work in the docker file's `RUN`.
I find it cleaner than running a single liner

Reply:
This should work:
```
    if [ "$JAVA_MAJOR_VERSION" == "8" ]; then \
        echo "Removing unnecessary JDK 8 binaries and libraries ..."; \
        rm -rf /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/javaws \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/*javafx* \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/*jfx* \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libdecora_sse.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libfxplugins.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libglass.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libgstreamer-lite.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libjavafx*.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libjfx*.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/libprism_*.so \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/deploy* \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/desktop \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/jfxrt.jar \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/javaws.jar \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/plugin.jar \
               /usr/lib/jvm/java-8-openjdk-amd64/jre/plugin \
               /usr/lib/jvm/java-8-openjdk-amd64/lib/*javafx*; \
    fi && \

    export PATH=$PATH:/docker-java-home && \

    echo "Done!"
```

Reply:
Done.
Turns out the main issue with the `if .. else` in Dockerfile was that I had to use `=` instead of `==` since it's runing with `sh`.